doc/source/api.rst

```diff
@@ -499,8 +499,10 @@ These can be accessed like ``Series.dt.<property>``.
    Series.dt.week
    Series.dt.weekofyear
```

Comment:
I think we should choose between `week` or `week_of_year` here. If we choose for `week` as the 'main' attribute, then I don't think it is needed to add `week_of_year`

But I don't really have a strong preference between `week` or `week_of_year`